File: template.py
from advent_of_code import *

input_value = get_input()
input_lines = get_input_lines()

class Problem:
    def __init__(self, problem, lines):
        self.problem = problem

    def run(self):
        return None

def run(*args):
    return Problem(1, *args).run()

def run2(*args):
    return Problem(2, *args).run()

example1 = multiline_lines(r"""
""")

run(example1) | eq()

run(input_lines) | debug('Star 1') | clipboard()

# run2(example1) | eq()

# run2(input_lines) | debug('Star 2') | clipboard()

File: advent_of_code/cpp/template.cpp
#include "../advent_of_code.hpp"

int run(int problem, vector<string> &lines) {
  // TODO
}

int main() {
  aocInit();

  auto inputLines = getInputLines();

  run(1, inputLines) | debug("Star 1") | eq(/* TODO */);
  run(2, inputLines) | debug("Star 2") | eq(/* TODO */);

  return aocExit();
}
